cmd/users/users.go

```diff
@@ -51,72 +59,57 @@ func run(cmd *cobra.Command, args []string) error {
 		log.Fatalf("Error: could not access 'output' flag - %s", err)
 	}
 
-	org, err := cmd.Flags().GetString(orgFlag)
-	if err != nil {
-		log.Fatalf("Error: could not access 'org' flag - %s", err)
-	}
+	ctx := context.Background()
 
 	// Github client
-	ctx := context.Background()
 	ts := oauth2.StaticTokenSource(
 		&oauth2.Token{AccessToken: token},
 	)
 	tc := oauth2.NewClient(ctx, ts)
-	client := github.NewClient(tc)
+	githubClient := github.NewClient(tc)
 
 	userExporter := export.New(export.Config{TargetPath: path + "/users.yaml"})
-	numUsers := 0
 
-	opt := &github.ListMembersOptions{
-		PublicOnly:  false,
-		ListOptions: github.ListOptions{PerPage: 100},
-	}
-	var allMembers []*github.User
-	for {
-		members, resp, err := client.Organizations.ListMembers(ctx, org, opt)
-		if err != nil {
-			log.Fatalf("Error: could not list organization members -- %v", err)
-		}
-		allMembers = append(allMembers, members...)
-		if resp.NextPage == 0 {
-			break
-		}
-		opt.Page = resp.NextPage
+	employees, err := personio.GetActiveEmployees(ctx, personioClientID, personioClientSecret)
```

Comment:
One sometimes encounters HTTP 501/502 with these API v1 endpoints.

It may make sense to implement retry here, if the surrounding task can't be rerun easily, might generate inconsistent data (PRs) or would yield error messages in such cases.

Reply:
I'll see how it behaves. If it fails, we get notified and can make the code more robust then.